crypto/keys/der/der.go

```diff
@@ -29,6 +29,21 @@ func FromProto(pb *keyspb.PrivateKey) (crypto.Signer, error) {
 	return UnmarshalPrivateKey(pb.GetDer())
 }
 
+// FromPublicProto takes a PublicKey protobuf message and returns the public
+// key contained within.
+func FromPublicProto(pb *keyspb.PublicKey) (crypto.PublicKey, error) {
+	return UnmarshalPublicKey(pb.GetDer())
+}
+
+// ToPublicProto returns a keyspb.PublicKey
```

Comment:
s/returns a keyspb.PublicKey/returns a keyspb.PublicKey that contains pubKey in DER encoding./